src/test/kotlin/com/igorwojda/integer/countdown/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countDown(n: Int): List<Int> {
-    TODO("not implemented")
+    return (0..n).reversed().toList()
```

Comment:
Ranges also have downTo so you don't wasted them creating a range and then reversing it.

Reply:
return (n downTo 0).toList()